tests/test_apps.py

```diff
@@ -44,7 +44,7 @@ def test_gemmini_matmul(golden):
     assert _test_app(module_file) == golden
 
 
-@pytest.mark.slow
+@pytest.mark.skip(reason="")
```

Comment:
Gemmini conv shouldn't be skipped

Reply:
My bad, I usually disable this test because it takes forever to run locally. Accidently, pushed that code.

Reply:
I think pytest took --fast option or sth like that to skip slow tests from the commandline